development/mimir-ingest-storage/config/grafana-agent.yaml

```diff
@@ -31,4 +37,4 @@ prometheus:
                 container: 'mimir-backend'
 
       remote_write:
-        - url: http://mimir-write-1:8080/api/v1/push
+        - url: http://mimir-write-zone-a-1:8080/api/v1/push
```

Comment:
Is this writing directly to ingester?

Reply:
Yes. In the local dev environment we don't have any balancer. Even if we have nginx, we don't do any load balancing via nginx.

Reply:
Oh, wait... my confusion comes from the fact that this is "read/write" deployment, so this endpoint is distributor's endpoint, not ingester's. (I thought we write to *ingester*, but that's not the case.)